.changeset/happy-feet-rhyme.md

```diff
@@ -0,0 +1,11 @@
+---
+"chainlink": minor
+---
+
+This PR introduce few changes:
+- Add a new flag support called `NewHeadsPollInterval` (0 by default indicate disabled), which is an interval for polling new block periodically using http client rather than subscribe to ws feed. 
```

Comment:
```suggestion
- Add a new config option `EVM.NodePool.NewHeadsPollInterval` (0 by default indicates disabled), which is an interval for polling new block periodically using http client rather than subscribe to ws feed. 
```

Reply:
The term `flag` might be confusing as it's used to refer to a config option specified in the command line (e.g. `chainlink node start --my-awesome-flag value`.
Also, we should specify the full path to the config option.